cpp/mrc/src/internal/control_plane/server.cpp

```diff
@@ -169,7 +169,7 @@ void Server::do_service_await_join()
 {
     // clear all instances which drops their held stream writers
     DVLOG(10) << "awaiting all streams";
-    m_connections.drop_all_streams();
```

Comment:
If this PR solves this part of the problem, we should remove `drop_all_streams` from the connection manangers interface.